src/platforms/unity/configuration/il2cpp.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: IL2CPP Line Numbers
+description: "Learn how the SDK provides line numbers to issues from IL2CPP builds."
+sidebar_order: 10
+---
+
+<Note>
+
+The IL2CPP line number support relies on the debug symbol upload to function properly. With the automated debug symbol upload enabled, the Sentry SDK will upload the line mapping through Sentry CLI.
+
+The feature is available from Unity versions `2020.3` and `2021.3` onwards. We running tests in CI against the current LTS, starting from `2020.3.37f1` and `2021.3.6f1`.
+
+</Note>
+
+Unity provides you the option to select [IL2CPP](https://docs.unity3d.com/Manual/IL2CPP.html) as scripting backend. This leads to C++ code being generated from your compiled C# (IL) code, hence the name `IL to CPP`.
+
+The Sentry SDK provides line number support for IL2CPP builds for the following platforms:
+
+- Desktop: Windows, macOS, Linux
+- Android
+- iOS
+
+When an error happens in your game, from its perspective, that errors happens in native code.
+The mapping between your C# code and the generated C++ code is part of the build process and does not get shipped with with your game. This leads to the event the Sentry SDK reports can only containing the native stacktrace. To be able to provide you with C# line numbers, the Sentry server needs to have access to that line mapping.
+
+The feature is currently experimental and opt-in through the editor configuration window `Advanced -> IL2CPP line numbers` or programmatically
+
+```csharp
+
+options.Il2CppLineNumberSupportEnabled = true;
+
+```
```

Comment:
You can remove the extra lines, the suggestion does not work with back tips.

Reply:
Thanks for the hint!